src/Microsoft.Sbom.Extensions/IMetadataBuilder.cs

```diff
@@ -41,4 +41,9 @@ public interface IMetadataBuilder
     /// Gets root package in JSON if supported.
     /// </summary>
     bool TryGetRootPackageJson(IInternalMetadataProvider internalMetadataProvider, out GenerationResult generationResult);
+
+    /// <summary>
+    /// Gets creation info in JSON if supported. Applies to only SPDX 3.0 and above.
```

Comment:
We don't know if this will be supported in SPDX 4.0
```suggestion
    /// Gets creation info in JSON if supported. Not available on SPDX 2.2.
```